content/blog/digidrug_seminar_series.md

```diff
@@ -0,0 +1,19 @@
+---
+title: SAVE THE DATE 09.09.21 - Digital Science for Drug Discovery
+date: 2021-08-10T16:00:00+02:00
```

Comment:
Ah it happens because it is posted in the future, right?

Reply:
I would set the date to the past, so that you can check the rendering locally.
![image](https://user-images.githubusercontent.com/7207093/128872493-304d145c-cbfd-4116-8f8a-ce2401da920f.png)

Only very small comment: I would add the key contacts as a list (not in one line).

Reply:
Must be the reason because it doesn't render for me either.

Reply:
FYI just tried changing the date to yesterday and the blog does render now. So, come tomorrow, it should be visible.

Reply:
With `hugo serve`, I mean.